vllm/attention/selector.py

```diff
@@ -168,6 +168,15 @@ def _cached_get_attn_backend(
             PlaceholderAttentionBackend)
         return PlaceholderAttentionBackend
     else:
+        # If the backend is not specified, it may be a plugin platform. Use the
```

Comment:
I think we should move `which_attn_to_use` function into `Platform`. can you update this pr in that direction?